src/backends/custom/custom.h

```diff
@@ -40,6 +40,14 @@ extern "C" {
 #define TRTIS_CUSTOM_EXPORT
 #endif
 
+#ifndef CUSTOM_VERSION
```

Comment:
I don't think we need this (see other comments). Instead we can just have the version be a uint32_t.